R/getbb.R

```diff
@@ -375,10 +375,8 @@ get_geotext_poly <- function (obj) {
     gt_p <- NULL
     indx <- which (!(seq (nrow (obj)) %in% indx_multi))
     index_final <- index_final [indx]
-    gt_p <- obj$geotext [indx] %>%
-        gsub ("POLYGON\\(\\(", "", .) %>%
-        gsub ("\\)\\)", "", .) %>%
-        strsplit (split = ",")
+    gt_p <- gsub ("(POLYGON\\(\\()|(\\)\\))", "", obj$geotext [indx])
```

Comment:
```suggestion
    ptn <- "(POLYGON\\(\\()|(\\)\\))"
    gt_p <- gsub (ptn, "", obj$geotext [indx])
```
(just to make the line shorter, and a bit easier to read)